From d0be507bbfa1c51a4aa57982962bcf3a6417b97e Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:10:58 -0400
Subject: [ethernet] i40evf: remove bogus comment

Message-id: <1409663591-32531-16-git-send-email-sassmann@redhat.com>
Patchwork-id: 90492
O-Subject: [RHEL7.1 PATCH 015/148] i40evf: remove bogus comment
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

This comment is simply not true.

Change-ID: If006b02b60984601a24257a951ae873dff568008
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
Tested-by: Sibai Li <sibai.li@intel.com>
Signed-off-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit db8ed10ff90c9a3c298f6c1ec635a1ac2503653c)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index db37ac4..1bb9deb 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -2048,7 +2048,6 @@ static void i40evf_init_task(struct work_struct *work)
         NETIF_F_HW_VLAN_CTAG_FILTER;
  }
 
- /* The HW MAC address was set and/or determined in sw_init */
  if (!is_valid_ether_addr(adapter->hw.mac.addr)) {
   dev_info(&pdev->dev, "Invalid MAC address %pMAC, using random\n",
     adapter->hw.mac.addr);
-- 
1.7.1